### PR TITLE
GOB: add ADI1 to detection tables

### PR DESCRIPTION
**Pull request by BJNFNE**
ADI1 can be started in ScummVM. 
ADI2.STK is is the start archive name same as in ADI2. But at this Game it handles about ADI 1 and not ADI 2 to avoid confuse i want to mention it here.

### CODE REVIEW

**Comment by BJNFNE** [comment]
Should be now applied. @sev-

File: engines/gob/gameidtotype.h [hunk 1]
```diff
@@ -69,6 +69,7 @@ static const GameIdToType gameIdToType[] = {
 	{                  "bambou", kGameTypeBambou },
 	{             "fascination", kGameTypeFascination },
 	{                  "geisha", kGameTypeGeisha },
+	{                    "adi1", kGameTypeNone },
```
Comment (bluegr) [inline engines/gob/gameidtotype.h:72]:
```suggestion
	{                    "adi1", kGameTypeAdi1 },
```
Also, please add kGameTypeAdi1 to the list of game types

Comment (BJNFNE) [reply engines/gob/gameidtotype.h]: @bluegr done

**Comment by bluegr** [comment]
Looks good now, thanks!

**Pull request merged**